spec/03-plugins/36-request-transformer/02-access_spec.lua

```diff
@@ -729,6 +731,24 @@ describe("Plugin: request-transformer(access) [#" .. strategy .. "]", function()
       local h_a_header = assert.request(r).has.header("x-a-header")
       assert.equals("true", h_a_header)
     end)
+    for _, seq in ipairs({ 1, 2, 3, 4, 5, 6}) do
+      it(fmt("override value if target header already exist: #%s", seq), function ()
+        local r = assert(client:send {
+          method = "GET",
+          path = "/request",
+          headers = {
+            host = "test9.test",
+            ["x-to-rename"] = "new-result",
+            ["x-is-renamed"] = "old-result",
```

Comment:
could you add a (regression) test that covers the bug you mentioned in this PR? for example a plugin configure to rename `x-to-rename` to `X-Is-Renamed`?

Reply:
Test added.